corehq/apps/case_importer/templates/case_importer/import_cases.html

```diff
@@ -142,10 +155,30 @@
                 <td class="col-md-1">
                     <!--ko text: user--><!--/ko-->
                 </td>
-                <td class="col-md-6" data-bind="with: task_status">
+                <td class="col-md-5" data-bind="with: task_status">
                     {% include 'case_importer/partials/ko_import_status.html' %}
                 </td>
-                <td class="col-md-2" data-bind="text: created"></td>
+                <td class="col-md-2">
+                    <!--ko if: upload_file_name-->
+                    <!--ko if: !upload_file_download_allowed-->
+                        <a style="cursor: pointer" data-bind="
+                            text: upload_file_name,
+                            popover: {
+                                title: $root.caseUploadPermissionDeniedTitle,
```

Comment:
Yeah this is changed in https://github.com/dimagi/commcare-hq/compare/case-import-history-phase2...case-import-history-phase-1-feedback, which I haven't PR'd yet. Sorry that's confusing.